src/main/java/net/fabricmc/loom/task/RemapTaskConfiguration.java

```diff
@@ -92,7 +92,7 @@ public void run() {
 		// Configure the default jar task
 		getTasks().named(JavaPlugin.JAR_TASK_NAME, AbstractArchiveTask.class).configure(task -> {
 			task.getArchiveClassifier().convention("dev");
-			task.getDestinationDirectory().set(new File(getProject().getBuildDir(), "devlibs"));
+			task.getDestinationDirectory().set(new File(getProject().getLayout().getBuildDirectory().getAsFile().get(), "devlibs"));
```

Comment:
You can make these lazy like so, removes the need to create it as a `File` as well.
```suggestion
			task.getDestinationDirectory().set(getProject().getLayout().getBuildDirectory().map(directory -> directory.dir("devlibs")));
```